share/spice/dota2/dota2.css

```diff
@@ -0,0 +1,49 @@
+.zci--dota2 .hero-image {
+    padding-left: -1em;
+    height: 306px;
+}
+
+.zci--dota2 .hero-image img {
+    height: 100%;
+    object-fit: cover;
```

Comment:
This will break in IE. We need an approach that will work for all major browsers, or at least fallback CSS for IE 9+